README.rst

```diff
@@ -12,6 +12,8 @@ the documentation locally if you have `giza
      cd docs/
      make html
 
+If successful, the built HTML files will be located in ``build/master/html/``.
```

Comment:
This isn't necessarily true; when we branch off stable releases, the build will happen under that branch's directory.

Reply:
Updated. Is this getting too messy now?